app/src/main/java/com/waseefakhtar/doseapp/feature/home/viewmodel/HomeViewModel.kt

```diff
@@ -56,4 +80,7 @@ class HomeViewModel @Inject constructor(
     fun getUserPlan() {
         // TODO: Get user plan
     }
+    companion object {
+        const val DATE_FILTER_KEY = "com.waseefakhtar.doseapp.MedicationDateFilter"
```

Comment:
Why don't we go for a simple `medication_date_filter` instead? :)

Reply:
You mean the key string?